foundations/installations/installing_node.md

```diff
@@ -38,18 +42,51 @@ You can verify `nvm` is installed by running the command:
 
 
 ~~~BASH
-command -v nvm
+command nvm -v
 ~~~
 
 if this returns `nvm: command not found` close the terminal and re-open it.
 
-On OS X you may continue to see the same message. If this happens, open your .bash_profile and add the following line of code:
+In macOS 10.15 and above, the default shell is zsh and nvm will look for .zshrc to update, none is installed by default.
+
+Simply run the following commands:
+
+~~~bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+~~~
+
+
+~~~bash
+touch ~/.zshrc
```

Comment:
If we `touch ~/.zshrc` before running the command to install nvm, we can skip the need to use nano to manually add the required `export ...` code as the nvm install script will find it automatically.

Reply:
I still think we should `touch ~/.zshrc` before curling the install script. It will save the user having to manually paste the export script.

Reply:
@xandora done 👍

Reply:
Thanks. :)